src/app/modules/item/pages/item-edit-advanced-parameters/item-edit-advanced-parameters.component.ts

```diff
@@ -90,23 +81,41 @@ export class ItemEditAdvancedParametersComponent implements OnInit, OnChanges {
     this.parentForm?.get('duration')?.updateValueAndValidity();
   }
 
+  onEnteringTimeMinEnabledChange(event: boolean): void {
+    if (!event) {
+      return;
+    }
+
+    this.parentForm?.get('entering_time_min')?.patchValue(
+      new Date()
+    );
+
+    this.computeEnteringTimeMaxValue();
+  }
+
   onEnteringTimeMaxEnabledChange(event: boolean): void {
     if (!event) {
       return;
     }
 
+    this.computeEnteringTimeMaxValue();
+  }
+
+  computeEnteringTimeMaxValue(): void {
+    const enteringTimeMinEnabled = this.parentForm?.get('entering_time_min_enabled')?.value as boolean;
     const enteringTimeMin = this.parentForm?.get('entering_time_min')?.value as string;
-    const enteringTimeMinDate = new Date(enteringTimeMin);
+    const enteringTimeMinDate = enteringTimeMinEnabled ? new Date(enteringTimeMin) : new Date();
     this.parentForm?.get('entering_time_max')?.patchValue(
       new Date(enteringTimeMinDate.getTime() + HOURS)
     );
   }
```

Comment:
Same here. I would say that when enabling max time, if it has its default value (`9999-12-31` I think), you leave this value, otherwise you set it `max(now, min time) + 1h`

Reply:
Fixed